src/containers/orders/order-history/order-history-item/order-history-item.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { getFormatDate } from '../../../../utils/date';
+import { getCurrencySign } from '../../../../utils/currency';
+import { statusColors } from '../../../../configs';
+import OrderHistoryTable from '../order-history-table/index';
+import OrderHistoryItemProduct from '../order-history-item-product';
+import { useStyles } from './order-history-item.styles';
+
+const OrderHistoryItem = ({ order }) => {
+  const { currency } = useSelector(({ Currency }) => ({
+    currency: Currency.currency
+  }));
+  const { t, i18n } = useTranslation();
+  const language = i18n.language === 'ua' ? 0 : 1;
```

Comment:
language is passing as a prop only to <OrderHistoryItemProduct /> , probably it can be deleted here and created in OrderHistoryItemProduct (if there are not a possibility to delete this ternary with language at all)

Reply:
yes, you`re right. it was used here earlier, but now it`s not. 
removed, thanks!